libp2p.nimble

```diff
@@ -8,16 +8,16 @@ license       = "MIT"
 skipDirs      = @["tests", "examples", "Nim", "tools", "scripts", "docs"]
 
 requires "nim >= 1.6.0",
-         "nimcrypto >= 0.4.1",
+         "nimcrypto >= 0.4.1 & < 1.0.0",
          "dnsclient >= 0.3.0 & < 0.4.0",
-         "bearssl >= 0.1.4",
-         "chronicles >= 0.10.2",
-         "chronos >= 3.0.6",
-         "metrics",
-         "secp256k1",
-         "stew#head",
-         "websock",
-         "unittest2"
+         "bearssl >= 0.1.4 & < 1.0.0",
+         "chronicles >= 0.10.2 & < 1.0.0",
+         "chronos >= 3.0.6 & < 4.0.0",
```

Comment:
why is this clamped for example? when going from clamped to non-clamped, you first need to verify that the newer version indeed is a problem (which it is not in several cases here) -  putting random version numbers here is a regression from the status quo